model/train_resnet.py

```diff
@@ -0,0 +1,445 @@
+#   Copyright (c) 2018 PaddlePaddle Authors. All Rights Reserved.
```

Comment:
@luotao1 perhaps will don't need to add new resnet.py. We can use existing ones to enable ngraph test.

Reply:
Yes, maybe you can use `python/paddle/fluid/tests/book/test_image_classification.py` for test